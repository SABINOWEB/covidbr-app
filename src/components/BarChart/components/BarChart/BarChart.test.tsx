import React from 'react'
import { BarChart } from './'
import { render } from '@testing-library/react'

describe('<BarChart />', () => {
  test('deve renderizar corretamente os dados', () => {
    const values = [
      {
        color: '#000000',
        totalValue: 100
      },
      {
        color: '#ffffff',
        totalValue: 50
      }
    ]

    const { container } = render(
      <BarChart
        data={values}
      />
    )
    expect(container).toMatchSnapshot()
  })
})
