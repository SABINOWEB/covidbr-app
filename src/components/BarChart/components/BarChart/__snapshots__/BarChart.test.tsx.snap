// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BarChart /> deve renderizar corretamente os dados 1`] = `
<div>
  <div
    class="sc-AxjAm jhiTqR"
  >
    <div
      class="sc-AxirZ fTEZtj"
      color="#000000"
    />
    <div
      class="sc-AxirZ fNDaIA"
      color="#ffffff"
    />
  </div>
</div>
`;
